inc/Engine/License/API/User.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace WP_Rocket\Engine\License\API;
+
+class User {
+	/**
+	 * The user object
+	 *
+	 * @var object
+	 */
+	private $user;
+
+	/**
+	 * Instantiate the class
+	 *
+	 * @param object $user The user object.
+	 */
+	public function __construct( $user ) {
+		$this->user = $user;
```

Comment:
```suggestion
		$this->user = is_object( $user ) ? $user : new stdClass;
```

This step eliminates all of the `is_object` checks in the class.